### ISSUE TITLE
IP configuration is not persistent on Disabling Interface and enabling back.

### ISSUE THREAD

**Issue by swe12345** [openbmc/phosphor-networkd#55]
Problem:
IP configuration is not persistent on Disabling Interface and enabling it back.

Steps to recreate:
1. Settings the interface as false using the curl command
curl -k -H "X-Auth-Token: $bmc_token" -D get.txt -X PATCH -d '{"InterfaceEnabled":false}'   https://$bmc/redfish/v1/Managers/bmc/EthernetInterfaces/eth0

2.Now check the configuration- it is lost- expected
3. Now Enable the interface back using curl commands
curl -k -H "X-Auth-Token: $bmc_token" -D get.txt -X PATCH -d '{"InterfaceEnabled":true}'   https://$bmc/redfish/v1/Managers/bmc/EthernetInterfaces/eth0

Expected Results:
IP configuration should persist

@sunharis @wak-google

**Comment by prkatti1** [comment]
I think changing the state of the interface should not have any impact on existing network settings.

**Comment by swe12345** [comment]
@wak-google Any update on this please

**Comment by wak-google** [comment]
This is expected, it refreshes the interface after it gets disabled which clears out all the IP info. We don't have a way to read IP information outside of what is configured on the interface.

**Comment by swe12345** [comment]
Disabling the interface, we should not loose the IP details right @wak-google??

**Comment by wak-google** [comment]
We do lose them today, AFAICT it will be difficult for us to model a change to this to preserve the details.

**Comment by sunharis** [comment]
> We do lose them today, AFAICT it will be difficult for us to model a change to this to preserve the details.

Yes. @swe12345 Can this issue be closed? This is working as per the design.
